stan/math/prim/mat/prob/ordered_logistic_lpmf.hpp

```diff
@@ -67,31 +75,180 @@ namespace stan {
       check_bounded(function, "Random variable", y, 1, K);
       check_finite(function, "Location parameter", lambda);
       check_greater(function, "Size of cut points parameter", c.size(), 0);
-      for (int i = 1; i < c.size(); ++i)
-        check_greater(function, "Cut points parameter", c(i), c(i - 1));
+      check_ordered(function, "Cut-points", c);
```

Comment:
Great!  Glad this was there.  Some of our original code really needs to be updated. When we add new utility code, like `check_ordered`, we don't do a good job of finding all the places where it should be used to simplify old code.